### PR TITLE
add Spritespin with git auto-update

### LINKED ISSUE #8721: [Request] Add spritespin
**Library name:** spritespin
**Git repository url:** https://github.com/giniedp/spritespin
**License(s):** https://github.com/giniedp/spritespin/blob/master/LICENSE
**Official homepage:** http://spritespin.ginie.eu/

### PR DESCRIPTION
**Pull request by kennynaoh**
PR for #8721 
@pvnr0082t, @x09326 please help me review this pull request, thank you.
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/giniedp/spritespin
 * Official website (optional, not the repository): http://spritespin.ginie.eu/index.html
 * NPM package url (optional):
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 236
   - Count of watchers: 18
   - Count of forks: 93
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #8721, cc @giniedp

### CODE REVIEW

File: ajax/libs/spritespin/package.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  ],
+  "author": "Alexander Graefenstein",
+  "license": "MIT",
+  "homepage": "http://spritespin.ginie.eu/index.html",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/giniedp/spritespin.git",
+    "basePath": "release",
+    "files": [
+      "spritespin*"
```
Comment (pvnr0082t) [inline ajax/libs/spritespin/package.json]: I think release is for distributed purpose just like dist. We can use `**/*`.

**Comment by pvnr0082t** [comment]
@PeterDaveHello Should we rename the filename of some old versions (i.e. jquery.spritespin.js
 -> spritespin.js )?

**Comment by PeterDaveHello** [comment]
@pvnr0082t In this case, nope.

**Comment by kennynaoh** [comment]
@pvnr0082t, @x09326 
I have fixed the autoupdate part.
Please help me review it again.
Thanks!

**Comment by pvnr0082t** [comment]
@PeterDaveHello I think this PR is ok. Please review it again, thank you.

**Comment by PeterDaveHello** [comment]
@kennynaoh notice the change of the title.

**Comment by kennynaoh** [comment]
Thanks!

**Comment by PeterDaveHello** [comment]
@kennynaoh no problem

**Pull request merged**